addons/stock/product_view.xml

```diff
@@ -248,12 +259,23 @@
                            type="object"  attrs="{'invisible':[('type', '=', 'service')]}" groups="stock.group_locations">
                         <field name="qty_available2" widget="statinfo"/>
                    </button>
-                   <button class="oe_stat_button" name= "action_view_stock_moves" type="object" attrs="{'invisible':[('type', '=', 'service')]}" string="Moves" groups="stock.group_stock_user" icon="fa-arrows-v"/>
-                   <button class="oe_stat_button" name="action_view_orderpoints" type="object" 
-                        attrs="{'invisible':[('type', 'in', ('service','consu'))]}" icon="fa-refresh">
+                    <button class="oe_stat_button" name= "action_view_stock_moves" type="object" attrs="{'invisible':[('type', '=', 'service')]}" string="Moves" groups= "stock.group_stock_user" icon="fa-arrows-v"/>
```

Comment:
Why is there an additional space at the start of the line and after the "groups=" now ?